static/css/homepage.scss

```diff
@@ -15,825 +15,27 @@
-        color: var(--gm-3-sys-light-primary);
-        display: flex;
-        flex-direction: row;
-        align-items: center;
-        font-size: 14px;
-        text-decoration: none;
-        .material-icons-outlined {
-          margin-right: 8px;
-          font-size: 18px;
-        }
-      }
-
-      &:hover {
-        box-shadow: var(--card-box-shadow-hover);
-        .topic-card-links a {
-          color: var(--gm-3-sys-light-primary-60);
-        }
-      }
-    }
-  }
-}
-
-.topic-card.clickable:hover {
-  cursor: pointer;
-}
-
-@include media-breakpoint-down(xs) {
-  #homepage #topics .topics-container {
-    gap: 10px;
-    margin-top: 32px;
-
-    .topic-card {
-      border-radius: 8px;
-      box-shadow: none;
-      height: 134px;
-      width: 166px;
-
-      .topic-card-body {
-        padding: 16px;
-      }
-
-      .topic-card-image-container {
-        height: 66px;
-        width: 132.5px;
-        // negative margins to account for scale transform below: (scaled - actual) / 2
-        margin-left: -66px;
-        margin-top: -32.5px;
-      }
-
-      .topic-card-image {
-        transform: scale(0.5);
-      }
-
-      .topic-card-title {
-        margin-top: 12px;
-        font-size: 16px;
-        font-weight: 500;
-        height: 24px;
-        line-height: 24px;
-        margin-bottom: 0px;
-        overflow: hidden;
-        text-align: center;
-        text-overflow: ellipsis;
-        white-space: nowrap;
-        width: 134px;
-      }
-
-      .topic-card-description,
-      .topic-card-links {
-        display: none;
-      }
-    }
-  }
-}
-
-// Styling for the partners carousel
-#homepage .carousel-container {
-  position: relative;
-  width: 100%;
-  overflow: hidden;
-
-  .carousel-items {
-    display: flex;
-    justify-content: flex-start;
-    align-items: stretch;
-    gap: 32px;
-    margin: 1rem;
-  }
-
-  a.carousel-item-container {
-    padding: 24px;
-    flex: 1 0 240px;
-    width: 0;
-
-    .image-container {
-      display: flex;
-      flex-grow: 1;
-      justify-content: center;
-      align-items: center;
-      margin-top: 0.5rem;
-
-      img {
-        max-width: 100%;
-        max-height: 100px;
-      }
-    }
-  }
-
-  .navigation-button {
-    position: absolute;
-    height: 100%;
-    top: 0;
-    right: 0;
-    display: flex;
-    align-items: center;
-    background-color: white;
-
-    .material-icons-outlined {
-      font-size: 1.5rem;
-      padding: 0.75rem;
-      border-radius: 2rem;
-      background-color: var(--dc-gray-lite);
-      color: white;
-      cursor: pointer;
-    }
-
-    .material-icons-outlined:hover {
-      box-shadow: $hover-box-shadow;
-    }
+@use "./new_variables" as var;
+@import "base";
+@import "./components/hero_video";
+@import "./components/sample_questions";
+@import "./components/tools";
+@import "./components/partners";
+@import "./components/build_your_own";
+@import "./components/topics";
+@import "./components/slide_carousel";
```

Comment:
also P2: would prefer if these were in a subfolder for these static or article style pages (as opposed to our data or topic explorer tools)